openpype/hosts/maya/plugins/publish/validate_rig_out_set_node_ids.py

```diff
@@ -24,6 +24,7 @@ class ValidateRigOutSetNodeIds(pyblish.api.InstancePlugin):
         openpype.hosts.maya.api.action.SelectInvalidAction,
         openpype.api.RepairAction
     ]
+    allow_history_only = False
```

Comment:
Good question.

Preserve backwards compatibility is essential, though in this case it is somehow a "bug fix", which can now be ignored if desired (for specific cases when have more than 1 shape).

So I feel like the default should be set to ```True```, even if it can break backwards compatibility...

But of course I will not make the decision :)

Reply:
> So I feel like the default should be set to True, even if it can break backwards compatibility...

Actually - `False` - right? Because when False the siblings without history are also considered? So the current implementation has the correct default. 👍